server/src/com/cloud/server/StatsCollector.java

```diff
@@ -285,9 +286,15 @@ private void init(Map<String, String> configs) {
                     s_logger.info(scheme + " is not a valid protocol for external statistics. No statistics will be send.");
```

Comment:
I realize it is out of scope for this PR, but would mind changing the log level to ``WARN`` for this message?  It is clearly a user intent that is not going to be carried through due to bad configuration data.